cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+import dateutil
+
+
+def advance_appliance_date_by_day(appliance):
+    txt_date = appliance.ssh_client.run_command('date --rfc-3339=ns').output
+    appliance_date = dateutil.parser.parse(txt_date)
+    td = datetime.timedelta(days=1)
+    advanced_txt_date = (appliance_date + td).strftime("%Y-%m-%d %H:%M:%S%z")
+    appliance.ssh_client.run_command("date -s '{}'".format(advanced_txt_date))
+
+
+def log_rotate_check(configured_appliance, log_path):
+    configured_appliance.ssh_client.run_command("/etc/cron.daily/logrotate")
+
+    initial_log_files = configured_appliance.ssh_client.run_command("ls -1 {}*".format(
+        log_path)).output.split('\n')
+    configured_appliance.ssh_client.run_command(
+        "echo 'Ensure line in logs' >> {}".format(log_path))
+    advance_appliance_date_by_day(configured_appliance)
+
+    # Perform the logrotate.
+    configured_appliance.ssh_client.run_command("/etc/cron.daily/logrotate")
```

Comment:
I would also consider asserting return code of this operation. It's just one line of code and might save some time during analysis when this test case fails. It's optional, though - I don't see it as required.

Reply:
I think the problem will be obvious from the traceback better than from any return code.